### DISCUSSION TITLE
View oldest deployments

### DISCUSSION THREAD

**Discussion by bscaspar** [vercel/vercel#7275] [Help] [answered]
I'm trying to do a "year in review" for my team, and want to look all the way back at our first deployment which happened just about a year ago. Is there an easy way to view deployments from that long ago? The `vercel ls` CLI command only displays 20 records at a time and the deployments page on my vercel account does the same, pulling 20 new records every time I hit the 'load more' button. 

I think we have over a thousand deployments over the past year so this doesn't seem like a great solution. Is there any way around this?

**Comment by Pieparker** [comment, marked as answer]
You could try the [API](https://vercel.com/docs/rest-api#endpoints/deployments/list-deployments)! There's a lot more flexibility with this approach.

**Reply by bscaspar** [reply]
Thanks! I ended up going a slightly different route - finding the commit I was looking for in Github and then opening the linked deployment - but this looks like it would work great too! Maybe slightly more configuration required to authenticate correctly.
